### DISCUSSION TITLE
Github Pages Link Down 404

### DISCUSSION THREAD

**Discussion by yetkinkrsngr** [community/community#64337] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

mY all github page go down 404 error code showing i just update my domin page all other page went down links still here but are not working :S

**Comment by GhostOf0days** [comment]
Try deploying with Netlify instead and make sure to have an _redirects page with '/* /index.html 200' in it.

**Comment by daneedev** [comment, marked as answer]
Hey,
make sure that you set correct domain in github pages settings, and that you used cname record ```yetkinkrsngr.github.io```.
